CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(Arbiter C CXX)
+
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Why not use a recent version, like 3.5 or 3.6.

Also, the docs say to put this first, even above the `project()` command.